ballerina/Ballerina.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
We do have a Ballerina.toml file with placeholders inside the build-config directory. Can you please check whether it is been used here?

We don't have to update the versions here every time. It should be automatically updated.

Reply:
Yes, this is the first commit of Ballerina.toml file.
This commit is to remove the `error: pathspec 'Ballerina.toml' did not match any file(s) known to git` from github workflow.